src/components/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import cn from 'classnames';
+import React, { useContext, useMemo } from 'react';
+import { TodoContext } from './TodoContext';
+import { Status } from '../types/Status';
+
+type Props = {};
+
+export const Footer: React.FC<Props> = React.memo(() => {
+  const {
+    filter,
+    setFilter,
+    todos,
+    setTodos,
+    setChecked,
+  } = useContext(TodoContext);
+
+  const handleSelected = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    const targetValue = event.target as HTMLElement;
+    const ourText = targetValue.innerText;
```

Comment:
Consider using interactive element as a target and set a value for it, to use event.target.value or event.target.name attribute here. It's not a best practice to use inner text. You may have the localization or different text formatting that can affect the performance